src/uct/api/uct_def.h

```diff
@@ -781,4 +781,18 @@ typedef ucs_status_t (*uct_tag_unexp_rndv_cb_t)(void *arg, unsigned flags,
                                                 const void *rkey_buf);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Callback to process asynchronous events.
+ *
+ * @param [in]  arg      User argument to be passed to the callback.
+ * @param [in]  flags    Flags to be passed to the callback (reserved for
+ *                       future use).
+ *
+ * @return @ref UCS_OK   - The asynchronous event was handled successfully.
+ *         Otherwise     - The asynchronous event was handled with an error.
```

Comment:
i think the return value should be void, UCT transport does not care if event was handler or not, it just reports it. same as with FD events.

Reply:
done